### PR TITLE
Generate default image when adding CoverFlow widget

### PR DESCRIPTION
**Pull request by DohyungLim**
- Adding the CoverFlow widget creates and runs the default image.

Signed-off-by: DoHyung Lim <delight.lim@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/changeable/common/coverflow.less [hunk 1]
```diff
@@ -1,5 +1,10 @@
+// CoverFlow style
 .ui-coverflow {
-	&:focus {
-		background-color: @color_focus_background;
-	}
-}
+	background-image: url("images/interactive/coverflow.png");
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/changeable/common/coverflow.less]: Widget looks little strange.
image

Comment (DohyungLim) [reply src/css/profile/mobile/changeable/common/coverflow.less]: I tested at WAPPL-VSC.
It worked fine.
How did you test the images you attached?

Comment (DohyungLim) [reply src/css/profile/mobile/changeable/common/coverflow.less]: Also, CoverFlow is only used for mobile profiles.

Comment (singa2000) [reply src/css/profile/mobile/changeable/common/coverflow.less]: @DohyungLim 
CoverFlow  is common profile and need to work in the wearable.
IAnd it also have to working the Tizen SDK Emulator.

**Comment by DohyungLim** [comment]
I will fix it in the Design-Editor.
TAU does not fix it.

sorry.

**Pull request closed without merging**